components/chunking/chunk-db-write-impl/src/test/kotlin/net/corda/chunking/db/impl/cpi/LiquibaseExtractorHelpersTest.kt

```diff
@@ -21,6 +21,7 @@ import java.io.FileNotFoundException
 import java.io.InputStream
 import java.nio.file.Files
 import java.nio.file.Path
+import java.util.*
```

Comment:
I think there is an `import` setting in IntelliJ that needs tweaking to prevent those wildcard imports.

```suggestion
import java.util.UUID
```

Reply:
Yeah I had turned it off but it was still sometimes doing it. I need to dig into that

Reply:
Might be under: `Editor->Code Style->Kotlin->Imports->Packages to Use import with '*'`
(it has been a while since I changed it)

Reply:
Yeah I had the Java one set, but not the kotlin one. I've both set properly now